proj/cmake/configure.cmake

```diff
@@ -103,14 +103,17 @@ if( CINDER_LINUX )
 elseif( CINDER_MAC )
 	set( CINDER_TARGET_SUBFOLDER "macosx" )
 elseif( CINDER_ANDROID )
-	set( CINDER_ANDROID_NDK_PLATFORM 21 CACHE STRING "Android NDK Platform version number." )
-	set( CINDER_ANDROID_NDK_ARCH "armeabi-v7a" CACHE STRING "Android NDK target architecture." )
-
-	set( CINDER_TARGET_SUBFOLDER "android-${CINDER_ANDROID_NDK_PLATFORM/CINDER_ANDROID_NDK_ARCH" )
+	#set( CINDER_ANDROID_NDK_PLATFORM 21 CACHE STRING "Android NDK Platform version number." )
```

Comment:
Don't we want to expose these variables to the cmake cache somehow?

Also, I do like that they are in `configure.cmake` as this file is shared by both libcinder and app builds.